hack/logextractor.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# This script can be used to extract the failure cluster logs tarball from the PVC created for an install pod.
+#
+# Usage: ./logextractor.sh [LOGS_PVC_NAME] [DEST_DIR]
```

Comment:
It would be nice if the argument was the name of the clusterdeployment instead of the PVC name. We should be able to find the PVC using labels, assuming the PVC is labeled with the clusterdeployment name.